pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>uk.gov.companieshouse</groupId>
     <artifactId>lfp-appeals-api</artifactId>
-    <version>0.0.1</version>
+    <version>0.0.2</version>
```

Comment:
I suggest we stop thinking about versions and releases for time being. I see no immediate value for us at this point. It matters for releases but only when CD is not enabled.

Reply:
I was just concerned about having multiple builds of the master branch under the same version and how the internal maven repo will handle this. But i know we need to get the CI/CD pipeline in first. I will revert this for now then